en-US/Characters_Codex/06_Spellcasting/Spell_Descriptions/H/Heal.md

```diff
@@ -25,7 +25,7 @@ _6th-level evocation_
 **Duration:** Instantaneous
 
 One creature that is neither a construct nor undead regains `70` hit points.
-The spell also ends blindness, deafness, and any diseases afflicting the target.
+The spell also ends any [deprived](#Conditions_deprived) and [impaired](#Conditions_impaired) conditions the creature may have and that are not caused by active environmental effects, and any diseases afflicting the target.
```

Comment:
Does this work for you?
The "...and that are not caused by active environmental effects" part is supposed to clarify that it doesn't counteract _lightly obscured_ or _heavily obscured_ environments, or things like the _Silence_ spell. I'm open to wording it differently though.